spec/datadog/core/configuration/components_spec.rb

```diff
@@ -1037,8 +1038,32 @@
           build_profiler
         end
 
+        context 'on Ruby 2.6 and above' do
+          before { skip 'Behavior does not apply to current Ruby version' if RUBY_VERSION < '2.6.' }
+
+          it 'logs a warning message mentioning that profiler has been force-enabled' do
+            expect(Datadog.logger).to receive(:warn).with(
+              /New Ruby profiler has been force-enabled. This feature is in alpha state/
+            )
+
+            build_profiler
+          end
+        end
+
+        context 'on Ruby 2.5 and below' do
+          before { skip 'Behavior does not apply to current Ruby version' if RUBY_VERSION >= '2.6.' }
+
+          it 'logs a warning message mentioning that profiler has been force-enabled AND that it may cause issues' do
+            expect(Datadog.logger).to receive(:warn).with(
+              /New Ruby profiler has been force-enabled on a legacy Ruby version \(< 2.6\). This is not recommended/
+            )
+
+            build_profiler
+          end
+        end
+
         context 'on Ruby 2.x' do
-          before { skip 'Behavior does not apply to current Ruby version' if RUBY_VERSION >= '3' }
+          before { skip 'Behavior does not apply to current Ruby version' if RUBY_VERSION >= '3.' }
```

Comment:
Why not `3.0`?

Reply:
Either is fine! Changed in 482d94643fc45c023482d38938380d85d3107e99